### ISSUE TITLE
Subgenera in TITAN

### ISSUE THREAD

**Issue by aoern** [CatalogueOfLife/data#558]
All the subgenera (>500) from TITAN have an infragenericEpithet that consists of the name of the genus and the name of the subgenus, for example Anaglyptus (Aglaophis).  As far as I understand, this is wrong and may lead to malfunctioning of client programs dealing with the epithet. All other sources produce infragenericEpithets consisting of the name of the subgenus only.

These subgenera have yet another oddity: There nameId is in 32-character UUID format. Earlier, I have seen UUID's only in conjunction with scientific names bad in a way or another (see #544).

**Comment by mdoering** [comment]
Actually it is the other way around. Subgenera should be given in the infragenericEpithet field, so they can be classiffied with a genus: https://github.com/CatalogueOfLife/coldp/blob/master/docs/publishing-guide.md#infrageneric-names

**Comment by mdoering** [comment]
... and yes, UUIDs are indeed indicators of a problem as they mean no stable ids

**Comment by yroskov** [comment]
For @gdower attention. It's related to my comment (https://github.com/CatalogueOfLife/testing/issues/231#issuecomment-1669873776)
>Multi Word Monomial, 520: 520 accepted subgenera recorded in combination with genus name.
https://www.checklistbank.org/catalogue/3/dataset/1032/workbench?facet=rank&facet=issue&facet=status&facet=nomStatus&facet=nameType&facet=field&facet=authorship&facet=authorshipYear&facet=extinct&facet=environment&facet=origin&issue=wrong%20monomial%20case&limit=800&offset=0
Examples: Abatocera (Sternobatocera) Breuning, 1943; Acalolepta (Pilohammus) Vitali, 2019; Bacchisa (Cyanastus) Pascoe, 1867
@gdower, it's not clear to me whether CLB able to handle subgenera in such form or not.

**Comment by mdoering** [comment]
> All the subgenera (>500) from TITAN have an infragenericEpithet that consists of the name of the genus and the name of the subgenus, for example Anaglyptus (Aglaophis). As far as I understand, this is wrong and may lead to malfunctioning of client programs dealing with the epithet. All other sources produce infragenericEpithets consisting of the name of the subgenus only.

This is wrong. A well formed subgenus should have the `genus` and the `infragenericEpithet`, not just the uninomial which I would guess can be found in some places _erroneously_.
